### ISSUE TITLE
[BUG] No options available to use cts.search options

### ISSUE THREAD

**Issue by JhansiBonthu** [marklogic-community/pipes#324]
As the current scenario, i need to use cts.search options like **wildcarded,stemmed,cts.indexOrder** etc.But i didnt see any option available over the pipes.

**To Reproduce**
1. Go to  Advanced Query 
2. Click on Query,we are able to place the cts.query which is required to perform the search.Haven't seen any option to place the search options.
Sample Query:
cts.search(
  (cts.andQuery([cts.collectionQuery("testCollection1"),
                cts.jsonPropertyRangeQuery("testId1", "=",   
                       cts.values(cts.elementReference(xs.QName("testId2")), null, null,
                                  cts.andQuery([cts.collectionQuery("testCollection2"),
                                  cts.jsonPropertyValueQuery("testId3","testdata"),   
                                  cts.jsonPropertyValueQuery("testId4", fn.string(var1))
                                               ])
                                 )
                                          )
                ])),
  [cts.indexOrder(cts.elementReference(xs.QName("Id")),"ascending")]

  )

3. Not only in advanced Query even in lookupQuery also there is no option available to add options.
for example : we don't have any option if i  want to add any options related to the cts.jsonpropertyValueQuery(e.g.,wildcarded)

**Expected behavior**
Looking for an option which we are able to add search options
and also for lookupQuery
**Screenshots**

**Desktop (please complete the following information):**

**Additional context**

**Comment by josvanroosmalen** [comment]
Yep we can do this but I am now focussed on getting the Javscript code generator working. Version 2.0 should be out within 1 or 2 weeks, so then I can pick this up.
